### PR TITLE
chore: adjust link rewriter

### PR DESCRIPTION
**Pull request by kendallstrautman**
## 🔗 Relevant links

- [Preview link](https://dev-portal-git-ksadjust-link-rewriter-hashicorp.vercel.app/) 🔎
- [Asana task](https://app.asana.com/0/1202276096167515/1202439144025004) 🎟️

## 🗒️ What

This PR adds additional docs pathnames to the docs path regex check to tighten up which docs links are rewritten for dev portal. 

## 🤷 Why

Fixing some broken links, particularly to the vault trial page. 

## 🧪 Testing

- [Visit this tutorial](https://dev-portal-git-ksadjust-link-rewriter-hashicorp.vercel.app/vault/tutorials/ops-pro-cert/ops-pro-overview) and click on the vault trial link
Screen Shot 2022-06-13 at 5 06 43 PM

## 💭 Anything else?

Many other links in the content were updated [in this PR](https://github.com/hashicorp/tutorials/pull/455)

### CODE REVIEW

File: src/lib/remark-plugins/rewrite-tutorial-links/__tests__/rewrite-tutorial-links.test.ts [hunk 1]
```diff
@@ -309,4 +315,33 @@ describe('rewriteTutorialLinks remark plugin', () => {
...
+
+  test('Beta product usecase path is not rewritten', async () => {
+    const contents = await remark()
+      .use(rewriteTutorialLinksPlugin)
+      .process(TEST_MD_LINKS.betaProductDocsLinkUseCases)
+
+    expect(String(contents)).toMatch(TEST_MD_LINKS.betaProductDocsLinkUseCases)
+  })
+
+  test('Beta product base domain path rewritten', async () => {
```
Comment (kendallstrautman) [inline src/lib/remark-plugins/rewrite-tutorial-links/__tests__/rewrite-tutorial-links.test.ts:335]:
```suggestion
  test('Beta product should only be determined by product dir, not tutorial name', async () => {
```

File: src/lib/remark-plugins/rewrite-tutorial-links/index.ts [hunk 2]
```diff
@@ -44,6 +44,13 @@ const PRODUCT_DOCS_PATHS = {
   vault: 'vaultproject.io',
   waypoint: 'waypointproject.io',
 }
+const ACCEPTED_DOCS_PATHNAMES = [
```
Comment (kendallstrautman) [inline src/lib/remark-plugins/rewrite-tutorial-links/index.ts:47]: Are there more I'm missing?

Comment (kendallstrautman) [inline src/lib/remark-plugins/rewrite-tutorial-links/index.ts:47]:
```suggestion
const ACCEPTED_DOCS_PATHS = [
```

**Review by BRKalow — APPROVED** [review]
👏

**Pull request merged**